Language/Functions/USB/Keyboard/keyboardModifiers.adoc

```diff
@@ -14,57 +14,82 @@ title: Keyboard Modifiers and Special Keys
+When given a printable ASCII character as an argument, the functions `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()` simulate actuations on the corresponding keys. These functions can also handle ASCII characters that require pressing a key in combination with Shift or, on international keyboards, AltGr. For example:
+[source,arduino]
+----
+Keyboard.write('a');  // press and release the 'A' key
+Keyboard.write('A');  // press Shift and 'A', then release both
+----
+A typical keyboard, however, has many keys that do not match a printable ASCII character. In order to simulate those keys, the library provides a set of macros that can be passed as arguments to `Keyboard.write()`, `Keyboard.press()` and `Keyboard.release()`. For example, the key combination Shift-F2 can be generated by:
+[source,arduino]
+----
+Keyboard.press(KEY_LEFT_SHIFT);  // press and hold Shift
+Keyboard.press(KEY_F2);          // press and hold F2
+Keyboard.releaseAll();           // release both
+----
+Note that, in order to press multiple keys simultaneously, one has to use link:../keyboardpress[`Keyboard.press()`] rather than link:../keyboardwrite[`Keyboard.write()`], as the latter just “hits” the keys (it presses and immediate releases them).
 [%hardbreaks]
-For more on ASCII values and the characters or functions they represent, see http://www.asciitable.com/[asciitable.com]
+The available macros are listed below:
 
 [float]
 === Keyboard modifiers
-A modifier key is a special key on a computer keyboard that modifies the normal action of another key when the two are pressed in combination.
-[%hardbreaks]
-For multiple key presses use link:../keyboardpress[Keyboard.press]()
-[%hardbreaks]
-The definitions of the modifier keys are listed below:
-[%hardbreaks]
-
+These are eight keys within the alphanumeric cluster of the keyboard that are meant to modify the normal action of another key when the two are pressed in combination. Whereas Shift and AltGr are used to access extra characters, the others are mostly used for keyboard shortcuts.
 
 |===
-|Key	|Hexadecimal value	|Decimal value
-
-|KEY_LEFT_CTRL  |0x80 |128
-|KEY_LEFT_SHIFT |0x81 |129
-|KEY_LEFT_ALT   |0x82 |130
-|KEY_LEFT_GUI   |0x83 |131
-|KEY_RIGHT_CTRL |0x84 |132
-|KEY_RIGHT_SHIFT |0x85 |133
-|KEY_RIGHT_ALT  |0x86 |134
-|KEY_RIGHT_GUI  |0x87 |135
+|Key	|Hexadecimal value	|Decimal value	|Notes
+
+|KEY_LEFT_CTRL  |0x80 |128 |
+|KEY_LEFT_SHIFT |0x81 |129 |
+|KEY_LEFT_ALT   |0x82 |130 |option (⌥) on Mac
+|KEY_LEFT_GUI   |0x83 |131 |OS logo, command (⌘) on Mac
+|KEY_RIGHT_CTRL |0x84 |132 |
+|KEY_RIGHT_SHIFT |0x85 |133 |
+|KEY_RIGHT_ALT  |0x86 |134 |also AltGr, option (⌥) on Mac
+|KEY_RIGHT_GUI  |0x87 |135 |OS logo, command (⌘) on Mac
```

Comment:
```suggestion
|KEY_LEFT_ALT   |0x82 |130 |Option (⌥) on Mac
|KEY_LEFT_GUI   |0x83 |131 |OS logo, Command (⌘) on Mac
|KEY_RIGHT_CTRL |0x84 |132 |
|KEY_RIGHT_SHIFT |0x85 |133 |
|KEY_RIGHT_ALT  |0x86 |134 |also AltGr, Option (⌥) on Mac
|KEY_RIGHT_GUI  |0x87 |135 |OS logo, Command (⌘) on Mac
```
Common convention is to capitalize key names.
For example:
https://github.com/github/docs/blob/main/contributing/content-style-guide.md#usage-highlights